.baedeker/init_fee.ts

```diff
@@ -0,0 +1,408 @@
+  const providerAssetHubC = new WsProvider(`${BDK_URL}/relay-assethubC/`);
+
+  const apiRelay = await ApiPromise.create({provider: providerRelay});
+  const apiAssetHubA = await ApiPromise.create({provider: providerAssetHubA});
+  const apiAssetHubB = await ApiPromise.create({provider: providerAssetHubB});
+  const apiAssetHubC = await ApiPromise.create({provider: providerAssetHubC});
+
+  const keyring = new Keyring({type: 'sr25519'});
+
+  const alice = keyring.addFromUri('//Alice');
+
+  await retry(() =>
+    apiRelay.tx.sudo
+      .sudo(apiRelay.tx.hrmp.forceOpenHrmpChannel(2001, 2002, 8, 8192))
+      .signAndSend(alice),
+  );
+
+  console.log('Hrmp channel for 2001 and 2002 opened!');
+  await new Promise(f => setTimeout(f, INTERVAL));
+
+  await retry(() =>
+    apiRelay.tx.sudo
+      .sudo(apiRelay.tx.hrmp.forceOpenHrmpChannel(2002, 2001, 8, 8192))
+      .signAndSend(alice),
+  );
+
+  console.log('Hrmp channel for 2002 and 2001 opened!');
+  await new Promise(f => setTimeout(f, INTERVAL));
+
+  await retry(() =>
+    apiRelay.tx.sudo
+      .sudo(apiRelay.tx.hrmp.forceOpenHrmpChannel(2001, 2003, 8, 8192))
+      .signAndSend(alice),
+  );
+
+  console.log('Hrmp channel for 2001 and 2003 opened!');
+  await new Promise(f => setTimeout(f, INTERVAL));
+
+  await retry(() =>
+    apiRelay.tx.sudo
+      .sudo(apiRelay.tx.hrmp.forceOpenHrmpChannel(2003, 2001, 8, 8192))
+      .signAndSend(alice),
+  );
+
+  console.log('Hrmp channel for 2003 and 2001 opened!');
+  await new Promise(f => setTimeout(f, INTERVAL));
+  await retry(() =>
+    apiRelay.tx.sudo
+      .sudo(apiRelay.tx.hrmp.forceOpenHrmpChannel(2003, 2002, 8, 8192))
+      .signAndSend(alice),
+  );
+
+  console.log('Hrmp channel for 2003 and 2002 opened!');
+  await new Promise(f => setTimeout(f, INTERVAL));
+  await retry(() =>
+    apiRelay.tx.sudo
+      .sudo(apiRelay.tx.hrmp.forceOpenHrmpChannel(2002, 2003, 8, 8192))
+      .signAndSend(alice),
+  );
+
+  console.log('Hrmp channel for 2002 and 2003 opened!');
+  await new Promise(f => setTimeout(f, 60000));
+
+  await retry(() =>
+    apiRelay.tx.sudo
+      .sudo(
+        apiRelay.tx.xcmPallet.send(
+          {
+            V4: {
+              parents: 0,
+              interior: {
+                X1: [
+                  {
+                    Parachain: 2001,
+                  },
+                ],
+              },
+            },
+          },
+          {
+            V4: [
+              {
+                UnpaidExecution: {
+                  weightLimit: 'Unlimited',
+                },
+              },
+              {
+                Transact: {
+                  originKind: 'Superuser',
+                  requireWeightAtMost: {
+                    refTime: 8000000000,
+                    proofSize: 8000,
+                  },
+                  call: {
+                    encoded:
+                      '0x3201011f00d43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d0104',
```

Comment:
make an extrinsic in code please

```suggestion
                      apiAssetHubA.tx.assets.forceCreate(1984, alice.address, true, 10000).method.toHex(),
```

Reply:
the same for the similar places